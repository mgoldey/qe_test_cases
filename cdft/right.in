&CONTROL
calculation =   "scf",
nstep=500,
outdir="out"
pseudo_dir="../pseudos/"
prefix='right'
/
&SYSTEM
ibrav = 0,
nat  =  2 ,
ntyp  =  1 ,
nbnd  =  30,
ecutwfc  =  60.0 ,
input_dft = "PBE"
assume_isolated='mt'
nspin = 2,
tot_magnetization = 1
tot_charge=1
/
&ELECTRONS
electron_maxstep = 1000
conv_thr    = 1.D-8,
diago_full_acc = .true.
/
&IONS
/
&CELL
/
K_POINTS gamma
ATOMIC_SPECIES
Zn 0.0 Zn_ONCV_PBE-1.0.UPF

CELL_PARAMETERS (angstrom)
10.0    0.0 0.0
0.0 10.0    0.0
0.0 0.0 15.0

ATOMIC_POSITIONS (angstrom)
Zn  5.0 5.0 5.0
Zn  5.0 5.0 10.0


EPCDFT
1 1e-6 4.0 20
delta_charge 2 2 1 1 -1.0 0.328

