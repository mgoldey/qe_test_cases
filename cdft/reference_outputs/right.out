
     Program PWSCF v.6.1 starts on 14Jun2017 at 17:50:20 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on       8 processor cores
     Number of MPI processes:                 8
     Threads/MPI process:                     1
     R & G space division:  proc/nbgrp/npool/nimage =       8
     Reading input from right.in
Warning: card &IONS ignored
Warning: card / ignored
Warning: card &CELL ignored
Warning: card / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     gamma-point specific algorithms are used
     Message from routine setup:
     the system is metallic, specify occupations

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         851     851    211                79440    79440    9908
     Max         854     854    214                79450    79450    9914
     Sum        6817    6817   1701               635539   635539   79289



     bravais-lattice index     =            0
     lattice parameter (alat)  =      18.8973  a.u.
     unit-cell volume          =   10122.5019 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =        39.00 (up:  20.00, down:  19.00)
     number of Kohn-Sham states=           30
     kinetic-energy cutoff     =      60.0000  Ry
     charge density cutoff     =     240.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)
     Assuming isolated system, Martyna-Tuckerman method


     celldm(1)=  18.897261  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.500000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  0.666667 )  


     PseudoPot. # 1 for Zn read from file:
     ../pseudos/Zn_ONCV_PBE-1.0.UPF
     MD5 check sum: cac91cab8b48577db0083da285b4abce
     Pseudo is Norm-conserving, Zval = 20.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1326 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2

     atomic species   valence    mass     pseudopotential
        Zn            20.00    65.39000     Zn( 1.00)

     Starting magnetic structure 
     atomic species   magnetization
        Zn           0.000

     16 Sym. Ops., with inversion, found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Zn  tau(   1) = (   0.5000000   0.5000000   0.5000000  )
         2           Zn  tau(   2) = (   0.5000000   0.5000000   1.0000000  )

     number of k points=     1
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

     Dense  grid:   317770 G-vectors     FFT dimensions: (  96,  96, 144)

     Estimated max dynamical RAM per process >      64.62MB

     Estimated total allocated dynamical RAM >     516.95MB
     Generating pointlists ...
     new r_m :   0.2063 (alat units)  3.8976 (a.u.) for type    1

     Initial potential from superposition of free atoms
     Check: negative starting charge=(component1):   -0.000355
     Check: negative starting charge=(component2):   -0.000355

     starting charge   39.99973, renormalised to   39.00000

     negative rho (up, down):  3.466E-04 3.466E-04
     Starting wfc are   20 randomized atomic wfcs +   10 random wfc

     total cpu time spent up to now is        3.9 secs

     per-process dynamical memory:    60.7 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations = 13.0

     negative rho (up, down):  5.005E-05 7.877E-05

     total cpu time spent up to now is       10.1 secs

     total energy              =    -798.58757687 Ry
     Harris-Foulkes estimate   =    -800.77927793 Ry
     estimated scf accuracy    <       1.89262137 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.00 Bohr mag/cell

     iteration #  2     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.85E-03,  avg # of iterations =  8.5

     negative rho (up, down):  1.615E-05 2.686E-05

     total cpu time spent up to now is       15.2 secs

     total energy              =    -799.20531243 Ry
     Harris-Foulkes estimate   =    -799.99750884 Ry
     estimated scf accuracy    <       2.17059794 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.01 Bohr mag/cell

     iteration #  3     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.85E-03,  avg # of iterations =  5.0

     negative rho (up, down):  2.060E-06 4.280E-06

     total cpu time spent up to now is       19.3 secs

     total energy              =    -799.76651041 Ry
     Harris-Foulkes estimate   =    -799.74825086 Ry
     estimated scf accuracy    <       0.03092282 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.06 Bohr mag/cell

     iteration #  4     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     WARNING:     2 eigenvalues not converged in regterg
     c_bands:  2 eigenvalues not converged
     WARNING:     4 eigenvalues not converged in regterg
     c_bands:  4 eigenvalues not converged
     ethr =  7.93E-05,  avg # of iterations = 20.0

     negative rho (up, down):  2.101E-06 3.693E-06

     total cpu time spent up to now is       27.3 secs

     total energy              =    -799.78185313 Ry
     Harris-Foulkes estimate   =    -799.78141510 Ry
     estimated scf accuracy    <       0.00779027 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.06 Bohr mag/cell

     iteration #  5     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     WARNING:     1 eigenvalues not converged in regterg
     c_bands:  1 eigenvalues not converged
     ethr =  2.00E-05,  avg # of iterations = 13.0

     negative rho (up, down):  3.029E-06 2.315E-06

     total cpu time spent up to now is       33.0 secs

     total energy              =    -799.78853844 Ry
     Harris-Foulkes estimate   =    -799.78409917 Ry
     estimated scf accuracy    <       0.00136103 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration #  6     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     WARNING:     1 eigenvalues not converged in regterg
     c_bands:  1 eigenvalues not converged
     ethr =  3.49E-06,  avg # of iterations = 15.0

     negative rho (up, down):  8.428E-12 1.735E-07

     total cpu time spent up to now is       39.1 secs

     total energy              =    -799.78661058 Ry
     Harris-Foulkes estimate   =    -799.78895139 Ry
     estimated scf accuracy    <       0.00047404 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration #  7     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.22E-06,  avg # of iterations = 12.5

     total cpu time spent up to now is       47.8 secs

     total energy              =    -799.78685216 Ry
     Harris-Foulkes estimate   =    -799.78685241 Ry
     estimated scf accuracy    <       0.00011349 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration #  8     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.91E-07,  avg # of iterations = 10.0

     negative rho (up, down):  2.341E-09 9.627E-07

     total cpu time spent up to now is       55.4 secs

     total energy              =    -799.78719431 Ry
     Harris-Foulkes estimate   =    -799.78687069 Ry
     estimated scf accuracy    <       0.00002140 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration #  9     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     WARNING:     1 eigenvalues not converged in regterg
     c_bands:  1 eigenvalues not converged
     ethr =  5.49E-08,  avg # of iterations = 13.0

     total cpu time spent up to now is       60.9 secs

     total energy              =    -799.78717561 Ry
     Harris-Foulkes estimate   =    -799.78720020 Ry
     estimated scf accuracy    <       0.00000134 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration # 10     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     WARNING:     1 eigenvalues not converged in regterg
     c_bands:  1 eigenvalues not converged
     ethr =  3.43E-09,  avg # of iterations = 15.5

     total cpu time spent up to now is       67.9 secs

     total energy              =    -799.78721157 Ry
     Harris-Foulkes estimate   =    -799.78717631 Ry
     estimated scf accuracy    <       0.00000008 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration # 11     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     WARNING:     1 eigenvalues not converged in regterg
     c_bands:  1 eigenvalues not converged
     ethr =  1.98E-10,  avg # of iterations = 18.0

     total cpu time spent up to now is       75.2 secs

     total energy              =    -799.78723114 Ry
     Harris-Foulkes estimate   =    -799.78721161 Ry
     estimated scf accuracy    <       0.00000010 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration # 12     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.98E-10,  avg # of iterations = 12.0

     total cpu time spent up to now is       80.6 secs

     total energy              =    -799.78724115 Ry
     Harris-Foulkes estimate   =    -799.78723118 Ry
     estimated scf accuracy    <       0.00000002 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration # 13     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  5.24E-11,  avg # of iterations =  9.0
       I D(charge)   A(charge)     Val=A-D      Target              Old              New
       1 0.101E+01   0.101E-01  -0.100E+01  -0.100E+01   0.32800000E+00   0.32768256E+00
     CDFT charge error:     0.79359228E-04

     Magnetic moment per site:
     atom:    1    charge:   18.9490    magn:    0.9509    constr:    0.0000
     atom:    2    charge:   19.8403    magn:    0.0050    constr:    0.0000

     total cpu time spent up to now is       86.5 secs

 ------ SPIN UP ------------


          k = 0.0000 0.0000 0.0000     band energies (ev):

  -136.4556-134.6326 -90.6195 -90.6112 -90.6112 -88.7326 -88.7326 -88.7311
   -17.3454 -17.3411 -17.3411 -17.3292 -17.3291 -15.4616 -15.4616 -15.4594
   -15.4594 -15.4587 -13.1788 -10.0325  -8.0905  -8.0905  -8.0084  -5.9348
    -4.9513  -4.6000  -4.4247  -4.2552  -4.2552  -3.8902

 ------ SPIN DOWN ----------


          k = 0.0000 0.0000 0.0000     band energies (ev):

  -136.4554-134.4673 -90.6194 -90.6112 -90.6112 -88.6733 -88.6733 -88.6716
   -17.3445 -17.3405 -17.3405 -17.3290 -17.3290 -15.2568 -15.2568 -15.2547
   -15.2547 -15.2540 -13.1731  -8.9472  -8.0867  -8.0867  -7.9721  -5.9243
    -4.9461  -4.5587  -4.4188  -4.2217  -4.2217  -3.7765

     highest occupied, lowest unoccupied level (ev):   -10.0325   -8.9472

     total energy              =    -799.78724116 Ry
     Harris-Foulkes estimate   =    -799.78724116 Ry
     estimated scf accuracy    <          1.7E-09 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration #  1     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     WARNING:     3 eigenvalues not converged in regterg
     c_bands:  3 eigenvalues not converged
     WARNING:     1 eigenvalues not converged in regterg
     c_bands:  1 eigenvalues not converged
     ethr =  5.24E-11,  avg # of iterations = 20.0
       I D(charge)   A(charge)     Val=A-D      Target              Old              New
       1 0.101E+01   0.102E-01  -0.100E+01  -0.100E+01   0.32768256E+00   0.32778971E+00
     CDFT charge error:    -0.40433032E-04

     Magnetic moment per site:
     atom:    1    charge:   18.9490    magn:    0.9509    constr:    0.0000
     atom:    2    charge:   19.8403    magn:    0.0050    constr:    0.0000

     total cpu time spent up to now is       94.6 secs

 ------ SPIN UP ------------


          k = 0.0000 0.0000 0.0000     band energies (ev):

  -136.4512-134.6369 -90.6152 -90.6069 -90.6069 -88.7370 -88.7370 -88.7354
   -17.3410 -17.3368 -17.3368 -17.3249 -17.3248 -15.4660 -15.4660 -15.4637
   -15.4637 -15.4630 -13.1745 -10.0367  -8.0862  -8.0862  -8.0045  -5.9308
    -4.9472  -4.5967  -4.4206  -4.2519  -4.2519  -3.8894

 ------ SPIN DOWN ----------


          k = 0.0000 0.0000 0.0000     band energies (ev):

  -136.4511-134.4716 -90.6151 -90.6069 -90.6069 -88.6777 -88.6777 -88.6759
   -17.3402 -17.3362 -17.3362 -17.3247 -17.3247 -15.2612 -15.2611 -15.2590
   -15.2590 -15.2583 -13.1688  -8.9512  -8.0825  -8.0825  -7.9683  -5.9203
    -4.9420  -4.5550  -4.4147  -4.2178  -4.2178  -3.7729

     highest occupied, lowest unoccupied level (ev):   -10.0367   -8.9512

     total energy              =    -799.78724116 Ry
     Harris-Foulkes estimate   =    -799.78724118 Ry
     estimated scf accuracy    <          3.7E-09 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration #  1     ecut=    60.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     WARNING:     2 eigenvalues not converged in regterg
     c_bands:  2 eigenvalues not converged
     ethr =  5.24E-11,  avg # of iterations = 16.0
       I D(charge)   A(charge)     Val=A-D      Target              Str
       1 0.101E+01   0.102E-01  -0.100E+01  -0.100E+01   0.32778971E+00
     CDFT charge error:    -0.34191173E-06

     Magnetic moment per site:
     atom:    1    charge:   18.9491    magn:    0.9509    constr:    0.0000
     atom:    2    charge:   19.8403    magn:    0.0050    constr:    0.0000

     total cpu time spent up to now is      101.8 secs

     End of self-consistent calculation

 ------ SPIN UP ------------


          k = 0.0000 0.0000 0.0000 ( 39645 PWs)   bands (ev):

  -136.4527-134.6354 -90.6167 -90.6083 -90.6083 -88.7355 -88.7355 -88.7339
   -17.3425 -17.3382 -17.3382 -17.3263 -17.3263 -15.4645 -15.4645 -15.4623
   -15.4623 -15.4616 -13.1759 -10.0353  -8.0876  -8.0876  -8.0058  -5.9321
    -4.9486  -4.5978  -4.4220  -4.2530  -4.2530  -3.8897

 ------ SPIN DOWN ----------


          k = 0.0000 0.0000 0.0000 ( 39645 PWs)   bands (ev):

  -136.4525-134.4702 -90.6165 -90.6083 -90.6083 -88.6762 -88.6762 -88.6744
   -17.3416 -17.3377 -17.3377 -17.3262 -17.3261 -15.2597 -15.2596 -15.2576
   -15.2576 -15.2569 -13.1703  -8.9499  -8.0839  -8.0839  -7.9696  -5.9217
    -4.9434  -4.5563  -4.4161  -4.2191  -4.2191  -3.7741

     highest occupied, lowest unoccupied level (ev):   -10.0353   -8.9499

!    total energy              =    -799.78726955 Ry
     Harris-Foulkes estimate   =    -799.78724117 Ry
     estimated scf accuracy    <          2.4E-09 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =   -1573.45453514 Ry
     hartree contribution      =     761.08392390 Ry
     xc contribution           =     -72.08501176 Ry
     ewald contribution        =      84.66835334 Ry
     CDFT contribution         =       0.00000011 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     convergence has been achieved in   1 iterations

     Writing output data file right.save

     init_run     :      3.48s CPU      3.81s WALL (       1 calls)
     electrons    :     93.55s CPU     97.93s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.71s CPU      0.77s WALL (       1 calls)
     potinit      :      1.88s CPU      2.01s WALL (       1 calls)

     Called by electrons:
     c_bands      :     65.18s CPU     68.00s WALL (      15 calls)
     sum_band     :      8.61s CPU      9.11s WALL (      15 calls)
     v_of_rho     :     15.66s CPU     16.49s WALL (      16 calls)
     mix_rho      :      3.08s CPU      3.32s WALL (      15 calls)

     Called by c_bands:
     init_us_2    :      0.33s CPU      0.48s WALL (      62 calls)
     regterg      :     64.40s CPU     66.84s WALL (      30 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :     50.23s CPU     52.20s WALL (     433 calls)
     g_psi        :      0.16s CPU      0.18s WALL (     401 calls)
     rdiaghg      :      1.14s CPU      1.17s WALL (     427 calls)

     Called by h_psi:
     h_psi:pot    :     49.65s CPU     51.63s WALL (     433 calls)
     h_psi:calbec :      2.11s CPU      2.17s WALL (     433 calls)
     vloc_psi     :     46.00s CPU     47.86s WALL (     433 calls)
     add_vuspsi   :      1.54s CPU      1.60s WALL (     433 calls)

     General routines
     calbec       :      2.11s CPU      2.16s WALL (     433 calls)
     fft          :     12.44s CPU     13.01s WALL (     470 calls)
     fftw         :     38.90s CPU     40.77s WALL (    4606 calls)

     Parallel routines
     fft_scatter  :     27.44s CPU     29.24s WALL (    5076 calls)

     PWSCF        :  1m37.23s CPU     1m41.99s WALL


   This run was terminated on:  17:52: 2  14Jun2017            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
